snapcraft_legacy/storeapi/v2/validation_sets.py

```diff
@@ -113,7 +115,7 @@ class BuildAssertion(EditableBuildAssertion):
     series: str
     """The "series" assertion header"""
 
-    sign_key_sha3_384: str | None
+    sign_key_sha3_384: str | None = None
```

Comment:
Given the `remove_sign_key` validator below, isn't this...

```suggestion
    sign_key_sha3_384: None = None
```